.github/json_matrices/build-matrix.json

```diff
@@ -14,23 +15,26 @@
         "ARCH": "arm64",
         "TARGET": "aarch64-unknown-linux-gnu",
         "PACKAGE_MANAGERS": ["pypi", "npm"],
-        "CONTAINER": "2_28"
+        "CONTAINER": "2_28",
+        "run": ["python", "node"]
```

Comment:
Please add java
```suggestion
        "run": ["python", "node", "java"]
```

Reply:
what about macos-12?

Reply:
Yes please. We need to test java client (all clients) on all 4 platform-arch combinations if label is set.